src/lib.rs

```diff
@@ -38,6 +38,38 @@
 //!
 //! This crate is guaranteed to compile on latest stable Rust. It *might* compile on older
 //! versions but that may change in any new patch release.
+//!
+//! # Examples
+//!
+//! ```
+//! use hash32::{FnvHasher, Hasher as _};
+//!
+//! #[derive(Hash)]
+//! struct Person {
+//!     id: u32,
+//!     name: &'static str,
+//!     phone: u64,
+//! }
+//!
+//! let person1 = Person {
+//!     id: 5,
+//!     name: "Janet",
+//!     phone: 555_666_7777,
+//! };
+//! let person2 = Person {
+//!     id: 5,
+//!     name: "Bob",
+//!     phone: 555_666_7777,
```

Comment:
```suggestion
//!     id: 6,
//!     name: "Bob",
//!     phone: 777_888_9999,
```
They should probably have a different `id` and data to make them more realistic. Did you leave the rest the same on purpose to demonstrate that the hash is different?

Reply:
I took the example from `std::hash`, which used the same ID and phone: https://doc.rust-lang.org/std/hash/index.html

I'm not sure why they used the same data/id.  I suspect you are correct that it is to demonstrate the hash is different.

I would like to keep the example as similar as possible, so that it can be used to compared to `std::hash`.

Reply:
Sounds good to me, thanks!